lib/iris/fileformats/netcdf.py

```diff
@@ -84,7 +84,7 @@
              'coordinates', '_FillValue', 'formula_terms', 'grid_mapping',
              'leap_month', 'leap_year', 'long_name', 'missing_value',
              'month_lengths', 'scale_factor', 'standard_error_multiplier',
-             'standard_name', 'units', 'valid_max', 'valid_min', 'valid_range']
```

Comment:
I'm just not convinced that removing these from the `_CF_ATTRS` list is a good idea... From looking at the [cf conventions document](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#attribute-appendix) it seems this is an appropriate place to keep them...